pkgs/racket-doc/scribblings/reference/vectors.scrbl

```diff
@@ -47,7 +47,20 @@ all elements, so for mutable data, mutating an element will affect other element
   v
 ]
 
-This function takes time proportional to @racket[size].}
+This function takes time proportional to @racket[size].
+
+A common mistake is using @racket[make-vector] to create nested vectors.
+The following creates a vector that contain 3 vectors to the same vector.
+@examples[
+  (make-vector 3 (vector)) ; wrong
```

Comment:
Strictly speaking, this is not "wrong", and you might sometimes want it... it's just that you often don't. Would you consider changing "wrong" to "probably not what you want" ?